src/factory.rs

```diff
@@ -397,6 +400,35 @@ impl Factory {
             target,
         }
     }
+
+    /// Create a basic mesh pipeline using a custom shader.
+    pub fn basic_pipeline(&mut self,
```

Comment:
Yeah, an example would be nice. But I know this works because I use it in https://github.com/kvark/plane-splitter/blob/59573c6d414d1b9952e168b73e3f2f5a6c72711c/src/main.rs#L35
So we can have example usage in follow-ups.